Removed ttl from dictionary response (#2709)
httpwg/http-extensions: draft-ietf-httpbis-compression-dictionary.md

normative:
  FOLDING: RFC8792
  HTTP: RFC9110

>   HTTP1.1: RFC2616

Reviewer: ```suggestion
  HTTP-CACHING: RFC9111
```

The Use-As-Dictionary response header is a Structured Field

> {{STRUCTURED-FIELDS}} sf-dictionary with values for "match", "match-dest",

Reviewer: ```suggestion
{{STRUCTURED-FIELDS}} with values for "match", "match-dest",
```

Currently practice is to NOT use the ABNF.

### Dictionary freshness requirement

> To be considered as a match, the dictionary resource MUST be either fresh
> [HTTP1.1] or within the "stale-while-revalidate" window defined in [RFC5861].

Reviewer: ```suggestion
To be considered as a match, the dictionary resource MUST be either fresh
[HTTP-CACHING] or allowed to be served stale (see eg [RFC5861]).
```

> [HTTP1.1]. Or within the "stale-while-revalidate" window defined in [RFC5861].

Reviewer 2: ```suggestion
[HTTP1.1] or within the "stale-while-revalidate" window defined in [RFC5861].
```
